lms/services/grading_info.py

```diff
@@ -23,27 +23,22 @@ def __init__(self, _context, request):
         self._db = request.db
         self._authority = request.registry.settings["h_authority"]
 
-    def get_by_assignment(self, oauth_consumer_key, context_id, resource_link_id):
+    def get_by_assignment(self, application_instance, context_id, resource_link_id):
         """
         Return all GradingInfo's for a given assignment.
 
         The returned list will contain one GradingInfo for each student who has
         launched this assignment (and had GradingInfo data persisted for them).
 
-        :param oauth_consumer_key: the assignment's oauth_consumer_key
+        :param application_instance: the assignment's application_instance
             (identifies a deployment of our app in an LMS)
-        :type oauth_consumer_key: str
-
```

Comment:
:+1: These are just clutter. We don't need so much documentation you can't see the code.